src/Renci.SshNet/SftpClient.cs

```diff
@@ -1704,7 +1705,7 @@ public string[] ReadAllLines(string path, Encoding encoding)
             {
                 while (!stream.EndOfStream)
                 {
-                    lines.Add(stream.ReadLine());
+                    lines.Add(stream.ReadLine()!);
```

Comment:
This block could be more canonically written as:

```c#
string? line;
while ((line = stream.ReadLine()) != null)
{
    lines.Add(line);
}
```

(github won't let me "add a suggestion")